docs/evaluate/development-features/interrupt-a-workflow.mdx

```diff
@@ -0,0 +1,41 @@
+---
+id: interrupt-workflow
+title: Interrupt a Workflow - Cancellation and Termination
+description: Learn how to manage Workflow interruptions in Temporal; understand how to gracefully handle Workflow cancellations and terminations to ensure proper cleanup and state management.
+sidebar_label: Interrupt a Workflow
+tags:
+  - workflow-interruption
+  - workflow-cancellation
+  - workflow-termination
+keywords:
+  - temporal workflow interruption
+  - cancel temporal workflows
+  - terminate temporal workflows
+  - temporal SDK cancellation
+  - workflow termination temporal
+  - temporal cancellation handling
+  - managing workflow state temporal
+  - temporal cleanup activities
+  - temporal workflow lifecycle
+  - temporal interruption strategies
+  - temporal SDK tutorial
+  - graceful workflow termination temporal
+  - temporal workflow management
+  - temporal activity interruption
+  - temporal termination best practices
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Discover how Temporal enables you to gracefully handle Workflow interruptions through cancellations and terminations.
+Understand how to stop a Workflow cleanly with cancellation, allowing for proper cleanup and state management, making it the preferred approach in most cases.
```

Comment:
What do you mean by "making it the preferred approach in most cases"? You mean making cancellations preferred over terminations? But there are times where you'd want to terminate a Workflow Execution too, which you describe on line 33 - maybe remove that part to make it more clear